safe-global/safe-wallet-web
# feat: recovery info modals + widgets

## What it solves

Resolves #2758 and #2759

## How this PR fixes it

This adds full-screen modals that appear if/when a recovery proposal is either added to the queue (for an owner/guardian) or when none is queued (for guardian).

## How to test it

1. Open a Safe with recovery enabled, ensuring no recovery transactions are queued. As a guardian, observe the "proposal" modal.
2. Open a Safe with recovery enabled, ensuring no recovery transactions are queued. As a owner/guardian, observe the "in-progress" modal.

## Screenshots

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/e515e926-0792-4e05-81c8-7c5a4a9372ac)

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/7dd562f2-c679-404f-a7c1-0172a70734ba)

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/c91800a3-f9a3-4d13-954e-48a0932509d2)

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/94756094-fae5-4994-a209-55045bb6484b)

## Checklist
